crates/ruff_python_formatter/src/expression/mod.rs

```diff
@@ -384,33 +387,45 @@ impl Format<PyFormatContext<'_>> for MaybeParenthesizeExpression<'_> {
         match needs_parentheses {
             OptionalParentheses::Multiline => match parenthesize {
                 Parenthesize::IfBreaksOrIfRequired => {
+                    let indent = !(f.context().options().preview().is_enabled()
+                        && is_expression_huggable(expression));
                     parenthesize_if_expands(&expression.format().with_options(Parentheses::Never))
+                        .with_indent(indent)
                         .fmt(f)
                 }
+
                 Parenthesize::IfRequired => {
                     expression.format().with_options(Parentheses::Never).fmt(f)
                 }
+
                 Parenthesize::Optional | Parenthesize::IfBreaks => {
                     if can_omit_optional_parentheses(expression, f.context()) {
                         optional_parentheses(&expression.format().with_options(Parentheses::Never))
                             .fmt(f)
                     } else {
+                        let indent = !(f.context().options().preview().is_enabled()
```

Comment:
Can you verify if it is necessary to set `indent` for all these cases? 

At least for `if [aaa, b]`, `can_omit_optional_parentheses` should always return `true` and I think the same is true for all other parenthesized expressions. Meaning, this change seems unnecessary. 

Can you test which changes are necessary and a) either remove the change or b) add (if not already) test cases proving that they are necessary

Reply:
Only one of these cases is necessary, but it felt odd to be inconsistent between branches -- like, we're then encoding details about which combinations of values are actually possible in practice, aren't we?